src/include/processor/execution_context.h

```diff
@@ -6,14 +6,23 @@
 namespace kuzu {
 namespace processor {
 
+class FactorizedTable;
+struct PopulatedCSVError;
+
 struct ExecutionContext {
     uint64_t queryID;
     common::Profiler* profiler;
     main::ClientContext* clientContext;
+    std::shared_ptr<FactorizedTable> warningTable;
+    uint64_t warningLimit;
```

Comment:
Can u wrap these two into a separate struct `WarningContext` or something along this line?

Reply:
updated, moved to `warning_context.h` (merged with `warning_schema.h`)